modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1685,9 +1670,6 @@ public String getCalendar(Opt<String> captureAgentId, Opt<String> seriesId, Opt<
       return cal.getCalendar().toString();
 
     } catch (Exception e) {
-      if (e instanceof SchedulerException)
-        throw e;
-      logger.error("Failed getting calendar: {}", getStackTrace(e));
```

Comment:
- `SchedulerException` is never thrown inside this block. Hence, the condition is always false.
- The exception does not need to be logged since it's thrown again anyway. Logging statements like this will just cause the same problem to be logged multiple times.